Options/WeakAurasOptions.lua

```diff
@@ -7148,14 +7162,48 @@ function WeakAuras.CreateFrame()
     frame.window = "default";
   end
 
+  local function extractTexts(input, ids)
+    local texts = {};
+
+    local currentPos, id, startIdLine, startId, endId, endIdLine;
+    while (true) do
+      startIdLine, startId = string.find(input, L["-- The following part is the\""], currentPos);
```

Comment:
So what I'm trying to say is, that I need the splitted comment here. And by reusing the same strings above, I actually ensure that they are translated the same. If I use %s and format, then translaters can mess up, and make the strings different, and thus break this feature. I think the best solution a string, where the id comes at the end.